wes/wes_output_API.json

```diff
@@ -1,330 +1,256 @@
 {
     "run id": [
+        [
+            "analysis/somatic/{run id}/{run id}_tnscope.output.vcf.gz",
+            "somatic variants: vcf file of somatic variants",
+            "VCF file of somatic variants using one of the following the Sentieon somatic callers {tnscope (default), tnhaplotyper2, tnsnv}.\n\nTNscope algorithm- https://support.sentieon.com/manual/usages/general/#tnscope-algorithm\nTNhaplotyper2- https://support.sentieon.com/manual/usages/general/#tnhaplotyper2-algorithm\nTNsnv - https://support.sentieon.com/manual/usages/general/#tnsnv-algorithm",
+            "somatic"
+        ],
+        [
+            "analysis/somatic/{run id}/{run id}_tnscope.output.maf",
+            "somatic variants: maf file of somatic variants",
+            "MAF file of VEP annotated variants using vcf2maf tool (https://github.com/mskcc/vcf2maf).  The vep annotated vcf (output.vcf.gz) file was used as the input for vcf2maf.",
+            "somatic"
+        ],
+        [
+            "analysis/somatic/{run id}/{run id}_tnscope.filter.vcf.gz",
+            "somatic variants: vcf file of filtered somatic variants",
+            "VCF file of filtered somatic variants.  With the output.vcf file as input, the vcftools software was used with parameter --remove-filtered-all to remove any variants whose FILTER column is anything other than PASS.  see http://vcftools.sourceforge.net/man_latest.html",
+            "somatic"
+        ],
+        [
+            "analysis/somatic/{run id}/{run id}_tnscope.filter.maf",
+            "somatic variants: maf file of filtered somatic variants",
+            "MAF file of VEP annotated filtered variants using vcf2maf tool (https://github.com/mskcc/vcf2maf).  The filtered vep annotated vcf file (filter.vep.vcf) file was used as input for vcf2maf.",
+            "somatic"
+        ],
+        [
+            "analysis/germline/{run id}/{run id}_vcfcompare.txt",
```

Comment:
[compare](https://github.com/CIMAC-CIDC/cidc-schemas/pull/369/files#r521478269) [cidc-schemas/wes/somatic.json#L47](https://github.com/CIMAC-CIDC/cidc-schemas/blob/wes-revamp2/cidc_schemas/schemas/assays/components/ngs/wes/somatic.json#L47) where vcf_compare is part of `somatic`

I know that this was moved from `germline` in the DM because that belongs to `normal`/`tumor` so there's two per analysis instead of the single file given.
Just want to double check that this is the correct solution before I add a special handler to template generation